x-pack/test/spaces_api_integration/spaces_only/apis/index.ts

```diff
@@ -11,11 +11,13 @@ export default function spacesOnlyTestSuite({ loadTestFile }: TestInvoker) {
   describe('spaces api without security', function() {
     this.tags('ciGroup5');
 
-    loadTestFile(require.resolve('./create'));
-    loadTestFile(require.resolve('./delete'));
-    loadTestFile(require.resolve('./get_all'));
-    loadTestFile(require.resolve('./get'));
-    loadTestFile(require.resolve('./select'));
-    loadTestFile(require.resolve('./update'));
+    loadTestFile(require.resolve('./copy_to_space'));
+    loadTestFile(require.resolve('./resolve_copy_to_space_conflicts'));
+    // loadTestFile(require.resolve('./create'));
```

Comment:
We'll need to uncomment these before merge.

Reply:
oops!